scripts/update-android-docs.sh

```diff
@@ -59,7 +59,7 @@ fi
 
 #Split version name to remove the "android-v" prefix
 array=(`echo $MAPS_SDK_VERSION | tr 'v' ' '` )
-MAPS_SDK_VERSION=${array[1]}
+MAPS_SDK_VERSION=${array[0]}
```

Comment:
basically we store version as `VERSION_NAME=10.0.0` - guess we can get rid of array completely here